corehq/apps/app_manager/id_strings.py

```diff
@@ -86,8 +86,10 @@ def _referral_detail_title_locale():
 
 
 @pattern('m%d.%s.title')
-def detail_title_locale(detail_type):
-    if detail_type.startswith('case') or detail_type.startswith('search'):
+def detail_title_locale(module, detail_type):
```

Comment:
Can't you remove these changes now that the translation is always available in the new display block?